.ci/component_descriptor

```diff
@@ -1,3 +1,17 @@
 #!/usr/bin/env bash
 
-"$(dirname $0)"/../vendor/github.com/gardener/gardener/hack/.ci/component_descriptor "$(dirname $0)"/..
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SOURCE_PATH="$(dirname $0)/.."
+
+"$SOURCE_PATH"/vendor/github.com/gardener/gardener/hack/.ci/component_descriptor "$SOURCE_PATH"
+
+echo "> Building component provider-aws"
+
+CA_PATH="$(mktemp -d)"
+mv "$COMPONENT_DESCRIPTOR_PATH" "$CA_PATH"
```

Comment:
the component descriptor fails with
```
.ci/component_descriptor: line 15: VERSION: unbound variable
```
So the version has to be read: `VERSION=$(cat $SOURCE_PATH/VERSION)`

Reply:
Talked to @AndreasBurger and the better approach is to use the `COMPONENT_VERSION ` env var.
The VERSION file is not overwritten in the `component-descriptor` step.

So then it would be:
```
component-cli ca "${CA_PATH}" "${CTF_PATH}" \
    -r  "$SOURCE_PATH"/.landscaper/resources.yaml \
    VERSION=${COMPONENT_VERSION}
```

Reply:
we have changed this to `VERSION=${EFFECTIVE_VERSION}`